src/pygama/pargen/energy_optimisation.py

```diff
@@ -1216,39 +1217,37 @@ def single_peak_fom(data, kwarg_dict):
     return out_dict
 
 
-def new_fom(data, kwarg_dict):
+def new_fom(data, kwarg_dict, alpha=None):
```

Comment:
I think adding an alpha here would break the dsp_optimize routines, would need checking

Reply:
default is None, so should not impact other routines.
I added this to allow calling `new_fom` having already an alpha value